index.html

```diff
@@ -1,25 +1,44 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <link rel="stylesheet" href="style.css">
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Capstone</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+    <title>Capstone</title>
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
   <nav class="nav-box">
```

Comment:
Great design @God-Logic 🤝

- [ ] Kindly ensure your design is a replica of the design provided. You may add the first navbar with the social icons to replicate the design given.
- Your design
![your-navbar](https://user-images.githubusercontent.com/63917725/233805331-cc5a1d70-f4c7-47e9-8098-f80c02ad0286.PNG)
- Given design
![given-navbar](https://user-images.githubusercontent.com/63917725/233805340-60ba972e-3877-4f6a-b293-9a9d39321479.PNG)